tasks/84_added-characters/added_characters.js

```diff
@@ -0,0 +1,9 @@
+
```

Comment:
This solution only returns first different element, but it should return three added characters.

It does not work for following examples:
string1 = "abcde" string2 = "db2a2ec2"
string1 = "hello" string2 = "eaaahllo"

Reply:
Please re-do